packages/babel-plugin-check-es2015-constants/test/fixtures/general/no-for-in/expected.js

```diff
@@ -0,0 +1,4 @@
+var MULTIPLIER = 5;
+throw new Error("\"MULTIPLIER\" is read-only");
+
+for (MULTIPLIER in arr) {}
```

Comment:
This should be
```js
var MULTIPLIER = 5;

for (MULTIPLIER in arr) {
  throw new Error("\"MULTIPLIER\" is read-only");
}
```

Because if arr is an empty object, `MULTIPLIER` is never reassigned and thus it shouldn't throw.